notebooks/219-pytorch-object-detection-accuracy-check-and-quantization/219-pytorch-object-detection-accuracy-check-and-quantization.ipynb

```diff
@@ -0,0 +1,578 @@
+   "id": "5c1e8029",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def get_config():\n",
+    "    config = Dict()\n",
+    "    data_yaml = check_yaml(\"./yolov5/data/coco128.yaml\")\n",
+    "    data = check_dataset(data_yaml)\n",
+    "\n",
+    "    model_config = Dict({\n",
+    "        \"model_name\": os.path.splitext(os.path.basename(\"./yolov5/yolov5m/yolov5m_openvino_model/yolov5m.xml\"))[0],\n",
+    "        \"model\": \"./yolov5/yolov5m/yolov5m_openvino_model/yolov5m.xml\",\n",
+    "        \"weights\": os.path.splitext(\"./yolov5/yolov5m/yolov5m_openvino_model/yolov5m.xml\")[0] + \".bin\"\n",
+    "    })\n",
+    "\n",
+    "    engine_config = Dict({\n",
+    "        \"device\": \"CPU\",\n",
+    "        \"stat_requests_number\": 8,\n",
+    "        \"eval_requests_number\": 8\n",
+    "    })\n",
+    "\n",
+    "    dataset_config = Dict({\n",
+    "        \"data_source\": data,\n",
+    "        \"imgsz\": 640,\n",
+    "        \"single_cls\": True,\n",
+    "    })\n",
+    "\n",
+    "    metric_config = Dict({\n",
+    "        \"conf_thres\": 0.001,\n",
+    "        \"iou_thres\": 0.65,\n",
+    "        \"single_cls\": True,\n",
+    "        \"nc\": 1 ,  # if opt.single_cls else int(data['nc']),\n",
+    "        \"names\": data[\"names\"],\n",
+    "        \"device\": \"cpu\"\n",
+    "    })\n",
+    "\n",
+    "    algorithms = [\n",
+    "        {\n",
+    "            \"name\": \"DefaultQuantization\",  # or AccuracyAwear\n",
+    "            \"params\": {\n",
+    "                    \"target_device\": \"CPU\",\n",
+    "                    \"preset\": \"performance\",\n",
+    "                    \"stat_subset_size\": 300,\n",
+    "                    \"model_type\": \"transformer\"\n",
+    "            }\n",
+    "        }\n",
+    "    ]\n",
+    "\n",
+    "    config[\"model\"] = model_config\n",
+    "    config[\"engine\"] = engine_config\n",
+    "    config[\"dataset\"] = dataset_config\n",
+    "    config[\"metric\"] = metric_config\n",
+    "    config[\"algorithms\"] = algorithms\n",
+    "    \n",
+    "    return config"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "608279bb",
+   "metadata": {
+    "id": "FAGmlKQ83ecE"
+   },
+   "source": [
+    "### Run Quantization Pipeline and Accuracy Verification\n",
+    "\n",
+    "Use 9 steps to quantize model by POT common function progress, the optimized model intermediate representation with collected min-max values will be saved. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "738cb5e7",
+   "metadata": {
+    "id": "QTOoQnSetzQM",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "config = get_config()  # Download dataset and set config\n",
+    "init_logger(level='INFO')\n",
+    "logger = get_logger(__name__)\n",
+    "save_dir = increment_path(Path(\"./yolov5/yolov5m/yolov5m_openvino_model/\"), exist_ok=True)  # increment run\n",
```

Comment:
Why use `increment_path` with `exist_ok=True`? That just returns the original pathname, as far as I understand.

Reply:
Just keep Ultralytics general usage for generated model saving, in case some user want to extract part code to merge into forked Ultralytics project.